src/main/java/net/mcreator/ui/workspace/AbstractMainWorkspacePanel.java

```diff
@@ -21,18 +21,27 @@
 
 import com.formdev.flatlaf.FlatClientProperties;
 import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.ComponentUtils;
+import net.mcreator.ui.init.L10N;
 import net.mcreator.ui.laf.themes.Theme;
+import net.mcreator.util.ColorUtils;
 
 import javax.annotation.Nullable;
 import javax.swing.*;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
 import java.awt.*;
+import java.awt.event.FocusAdapter;
+import java.awt.event.FocusEvent;
 import java.util.HashMap;
 import java.util.Map;
 
 public abstract class AbstractMainWorkspacePanel extends JPanel {
 
 	protected final MCreator mcreator;
 
+	public final JTextField search;

```

Comment:
I would recommend making this not public and instead making the method getSearch or maybe better even getSearchTerm.

Reply:
Code errors dialog would then need `setSearchTerm()` as well.

Reply:
:heavy_check_mark: Fixed